poc/docs/Zeppelin_ODFE_Storage.md

```diff
@@ -26,42 +26,57 @@
 
 ### **Usage:**
 
-1. POC for ODFE adapter is stored in branch 'zeppelin-odfe' of kibana-notebooks
+
+1. Clone kibana-notebooks
+
 ```
-git checkout zeppelin-odfe
+git clone https://github.com/opendistro-for-elasticsearch/kibana-notebooks.git
```

Comment:
generify the example command / add ssh ?

Reply:
Thanks, removed the clone commands and added link to repositories.